Centaurus/Alpha/AlphaStartup.cs

```diff
@@ -0,0 +1,165 @@
+        {
+            return Host.CreateDefaultBuilder()
+                .ConfigureLogging(logging =>
+                {
+
+                    logging.ClearProviders();
+                    logging.AddConsole();
+
+                    if (settings.Verbose)
+                        logging.SetMinimumLevel(Microsoft.Extensions.Logging.LogLevel.Trace);
+                    else if (settings.Silent)
+                        logging.SetMinimumLevel(Microsoft.Extensions.Logging.LogLevel.Error);
+                    else
+                        logging.SetMinimumLevel(Microsoft.Extensions.Logging.LogLevel.Information);
+                })
+                .UseNLog()
+                .ConfigureWebHostDefaults(webBuilder =>
+                {
+                    webBuilder.UseStartup<Startup>()
+                        .UseUrls(settings.AlphaUrl);
+                });
+        }
+
+        class Startup
+        {
+            public Startup(IConfiguration configuration)
+            {
+                Configuration = configuration;
+            }
+
+            public IConfiguration Configuration { get; }
+
+            // This method gets called by the runtime. Use this method to add services to the container.
+            // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
+            public void ConfigureServices(IServiceCollection services)
+            {
+                services
+                    .AddMvc(options => options.EnableEndpointRouting = false);
+                //.AddJsonOptions(options =>
+                //{
+                //    options.JsonSerializerOptions.Converters.Add(new AssetSettingsConverter());
+                //});
+
+                services.Add(
+                    new ServiceDescriptor(
+                        typeof(IActionResultExecutor<JsonResult>),
+                        Type.GetType("Microsoft.AspNetCore.Mvc.Infrastructure.SystemTextJsonResultExecutor, Microsoft.AspNetCore.Mvc.Core"),
+                        ServiceLifetime.Singleton)
+                );
+            }
+
+            static ApplicationState[] ValidApplicationStates = new ApplicationState[] { ApplicationState.Rising, ApplicationState.Running, ApplicationState.Ready };
+
+            // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+            public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+            {
+                if (env.IsDevelopment())
+                {
+                    app.UseDeveloperExceptionPage();
+                }
+                else
+                {
+                    app.UseExceptionHandler("/Error");
+                    // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
+                    app.UseHsts();
+                }
+
+                app.UseStaticFiles();
```

Comment:
Do we need this? What static files do we server?

Reply:
No we don't. I will remove it